### PR TITLE
Remove unused module from doc's example.

### PR DESCRIPTION
**Pull request by ar4s**
This PR remove ``models`` from example in docs.

### CODE REVIEW

**Comment by timgraham** [comment]
Could you do a more comprehensive check? I also spotted that in `howto/writing-migrations` and `topics/migrations.txt`.

**Comment by ar4s** [comment]
Ok, I checked also ``howto/writing-migrations.txt`` and ``topics/migrations.txt``

File: docs/topics/migrations.txt [hunk 1]
```diff
@@ -455,7 +455,7 @@ the file in the right place, suggest a name, and add dependencies for you)::
 Then, open up the file; it should look something like this::
 
     # Generated by Django A.B on YYYY-MM-DD HH:MM
-    from django.db import migrations, models
+    from django.db import migrations
```
Comment (timgraham) [inline docs/topics/migrations.txt]: Isn't this correct for the auto-generated migration?

Comment (ar4s) [inline docs/topics/migrations.txt]: It is correct for ``makemigrations`` with ``--empty`` switch.

Comment (timgraham) [inline docs/topics/migrations.txt]: Thanks, I forgot the behavior changed in a7bc00e17bebe34cdf23f744937b3c386ee73d81.

**Pull request merged**